nanome_minimization/Minimization.py

```diff
@@ -68,10 +70,11 @@ def minimization_done(self):
             self.__integration_request.send_response(True)
 
     def set_run_status(self, running):
+        btn_type = nanome.util.enums.PluginListButtonType.run
         if running:
-            self.set_plugin_list_button(nanome.util.enums.PluginListButtonType.run, "Stop")
+            self.set_plugin_list_button(btn_type, "Stop")
         else:
-            self.set_plugin_list_button(nanome.util.enums.PluginListButtonType.run, "Run")
+            self.set_plugin_list_button(btn_type, "Run")
```

Comment:
I like the `btn_type` here, could make it even better with inline if:
```
self.set_plugin_list_button(btn_type, "Stop" if running else "Run")
```
(GitHub won't let me do suggestion block here)